src/App.jsx

```diff
@@ -0,0 +1,86 @@
+import React, {useState, useEffect} from 'react';
+import ReactDOM from 'react-dom';
+import GameRow from './components/GameRow.jsx';
+
+const dumbRow = [0, 0, 0, 0, 0, 0, 0, 0, 0, 0];
+
+const App = () => {
+
+    let startingBoard = [
+        [0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
+        [0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
+        [0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
+        [0, 0, 0, 1, 1, 1, 0, 0, 0, 0],
+        [0, 0, 0, 0, 0, 1, 0, 0, 0, 0],
+        [0, 0, 0, 0, 1, 0, 0, 0, 0, 0],
+        [0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
+        [0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
+        [0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
+        [0, 0, 0, 0, 0, 0, 0, 0, 0, 0]
+    ];
+
+    const assessBoard = (board) => {
+        let nextBoard = new Array(startingBoard.length).fill(0);
+        const len = board.length;
+
+        nextBoard = nextBoard.map( () => []);
+
+        console.log(nextBoard);
```

Comment:
console.logs are good for debugging but you have to avoid pushing them to you repo